src/Google.Maps/Google.Maps.csproj

```diff
@@ -1,6 +1,6 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFrameworks>net461;netstandard1.3</TargetFrameworks>
+    <TargetFrameworks>netstandard2.0</TargetFrameworks>

```

Comment:
unfortunately this change makes it much harder to approve the pull request.  I would consider changing the targetframework to be considered out of scope of adding traffic_model to the request.